web/html/src/manager/admin/access-group/access-group-list.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from "react";
+
+import { Button } from "components/buttons";
+import { TopPanel } from "components/panels/TopPanel";
+import { Column } from "components/table/Column";
+import { SearchField } from "components/table/SearchField";
+import { Table } from "components/table/Table";
+
+import { Utils } from "utils/functions";
+
+const dataTest = {
+  items: [
+    {
+      id: 1000010000,
+      name: "Content Management",
+      description: "View image details, patches, packages, build log and cluster information",
+      type: "Built-in",
+      users: 0,
+      permissions: 15,
+      selected: false,
+      disabled: false,
+    },
+    {
+      id: 1000020000,
+      name: "Activation Key Administrator",
+      description: "View Activation Administrator, Configuration Administrator.",
+      type: "Built-in",
+      users: 2,
+      permissions: 5,
+      selected: false,
+      disabled: false,
+    },
+    {
+      id: 1000030000,
+      name: "Configuration Administrator",
+      description: "View Configuration Administrator Administrator, Configuration Administrator.",
+      type: "Built-in",
+      users: 4,
+      permissions: 15,
+      selected: false,
+      disabled: false,
+    },
+    {
+      id: 1000040000,
+      name: "System Group Administrator",
+      description: "View Activation Administrator, Configuration Administrator.",
+      type: "Built-in",
+      users: 2,
+      permissions: 5,
+      selected: false,
+      disabled: false,
+    },
+    {
+      id: 1000050000,
+      name: "Custom group1",
+      description: "Custom group1 Activation Administrator, Configuration Administrator.",
+      type: "Custome",
+      users: 7,
+      permissions: 25,
+      selected: false,
+      disabled: false,
+    },
+  ],
+  total: 19,
+  selectedIds: [],
+};
+
+const actionButtons = (type) => {
+  if (type === "Built-in") {
+    return (
+      <div className="btn-group">
+        <Button className="btn-default btn-sm" icon="fa-user" />
+      </div>
+    );
+  } else {
+    return (
+      <div className="btn-group">
+        <Button className="btn-default btn-sm" icon="fa-user" />
+        <Button className="btn-default btn-sm" icon="fa-pencil" />
+        <Button className="btn-default btn-sm" icon="fa-trash" />
+      </div>
+    );
+  }
+};
+export function AccessGroupList(props) {
+  const addAccessGroup = () => {
+    window.pageRenderers?.spaengine?.navigate?.(`/rhn/manager/admin/access-group/create`);
```

Comment:
I would recommend against this, if the only thing a button does is be a link, it should just be a link element, even if it's a button visually. That way the user gets all the default behaviors from a browser, it works with accessibility, etc. The same point applies for `addAccessGroup`, for `onCancel`, etc, basically everywhere where we use this pattern right now.